cli/api/backup.go

```diff
@@ -29,13 +30,17 @@ func (a *api) Backup(dirpath string, excludes []string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-
 	var path string
 	req := dao.BackupRequest{
 		Dirpath:              dirpath,
 		SnapshotSpacePercent: config.GetOptions().SnapshotSpacePercent,
 		Excludes:             excludes,
 	}
+
+	est := dao.BackupEstimate{}
```

Comment:
I tried earlier, but there were dependencies (BackupRequest, specifically) that caused problems for the other backup functions here when I tried to move it. Per our discussion, I'd like to hold off on moving this code until Alderaan is released.